WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -216,12 +349,125 @@ public void onResponse(JSONObject response) {
                 }, new RestRequest.ErrorListener() {
                     @Override
                     public void onErrorResponse(VolleyError error) {
-                        AppLog.e(AppLog.T.API, "Error response to Jetpack Settings REST request: " + error);
+                        AppLog.w(AppLog.T.API, "Error fetching Jetpack Monitor module options: " + error);
                         notifyUpdatedOnUiThread(error);
                     }
                 });
     }
 
+    private void pushJetpackProtectSettings() {
+        WordPress.getRestClientUtils().setJetpackProtect(
+                mSite.getSiteId(), mJpSettings.jetpackProtectEnabled, new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        mRemoteJpSettings.jetpackProtectEnabled = response.optBoolean("active");
+                        mJpSettings.jetpackProtectEnabled = mRemoteJpSettings.jetpackProtectEnabled;
+                        String status = mJpSettings.jetpackProtectEnabled ? "activated" : "deactivated";
+                        AppLog.d(AppLog.T.API, "Jetpack Protect module " + status);
+                        notifySavedOnUiThread(null);
+                    }
+                }, new RestRequest.ErrorListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                        AppLog.w(AppLog.T.API, "Error updating Jetpack Protect module: " + error);
+                    }
+                });
+    }
+
+    private void pushJetpackMonitorSettings() {
+        WordPress.getRestClientUtils().setJetpackMonitor(
+                mSite.getSiteId(), mJpSettings.monitorActive, new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        mRemoteJpSettings.monitorActive = response.optBoolean("active");
+                        mJpSettings.monitorActive = mRemoteJpSettings.monitorActive;
+                        String status = mJpSettings.jetpackProtectEnabled ? "activated" : "deactivated";
+                        AppLog.d(AppLog.T.API, "Jetpack Monitor module " + status);
+                        notifySavedOnUiThread(null);
+                    }
+                }, new RestRequest.ErrorListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                        AppLog.w(AppLog.T.API, "Error updating Jetpack Monitor module: " + error);
+                        notifySavedOnUiThread(error);
```

Comment:
This is a bit inconsistent between calls. Some of the errors have this call and most don't. Unless there is a reason, we should be consistent everywhere. If there is a reason, I think it's worth a comment when it's required.